### PR TITLE
Avatar 

### PR DESCRIPTION
**Pull request by mehulsatardekar**
Added Avatar Components

Preview :
https://framer-ui.netlify.app/docs/avatar.html

### CODE REVIEW

**Comment by rahulyadav139** [comment]
Everything is great. Go ahead to merge it.

**Comment by rahulk31** [comment]
UI is great and Code Looks flawless to me, ready to merge.

File: components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+ }
+ .avatar-md{
+     width: 7rem;   
+ }
+ .avatar-sm{
+     width: 5rem;   
+ }
+ .avatar-ex-sm{
+     width: 3rem;
+ }
```
Comment (Sahil25061999) [inline components/avatar/avatar.css:24]: code looks ready to merge, just a suggestion naming can be shorten.

Comment (mehulsatardekar) [reply components/avatar/avatar.css:24]: @Sahil25061999 I appreciate your suggestion can you elaborate more I mean what naming part can be reduced, here I am already using shorthand naming for extra-small.

File: components/avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
+/* @import url('../../components/root.css'); */
```
Comment (viru2001) [inline components/avatar/avatar.css]: you can remove this commented code

**Review by viru2001 — COMMENTED** [review]
UI looks Good and also website is responsive that's great.

**Review by rahulyadav139 — APPROVED** [review]

**Review by viru2001 — APPROVED** [review]

**Pull request merged**